src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -128,6 +130,7 @@ const ProductsTable: FunctionComponent<ProductsTableProps> = ({
   ]);
   const removeFilter = () => {
     setFilter('');
+    navigate('/subscriptions/inventory');
```

Comment:
Suggestion: This change is not needed (and may cause weird behavior) as the filtering logic was all handled in TEAMNADO-7112

Reply:
I think this file in general shouldn't need to change